### PR TITLE
Singular CTA with plausible events

### PR DESCRIPTION
**Pull request by goodroot**
This PR does a few things:

1. Adds a CTA button to replace existing ActionFooter
2. Allows `buttonText` to be set within frontmatter of blog or default
3. Increases font size as per discussions
4. Changes how headers descend in size and weight
5. Sets up Custom Event tracking in plausible: https://plausible.io/docs/custom-props/for-custom-events

Frontmatter example:

```markdown
---
title: "QuestDB Enterprise: Role-based Access Control Walkthrough"
author: QuestDB Team
author_title: The Makers of QuestDB
author_url: https://github.com/questdb
author_image_url: https://avatars.githubusercontent.com/u/52297642?s=200&v=4
description: Role-based access control is now available in QuestDB Enterprise. This article presents a walkthrough of a basic implementation.
tags: [tutorial, rbac, enterprise]
image: /img/blog/2023-08-15/roles.webp
slug: enterprise-rbac-access-control-list-tutorial
buttonText: "Try ACLs on QuestDB Cloud"
---
```

### CODE REVIEW

File: src/theme/BlogPostPage/index.tsx [hunk 1]
```diff
@@ -1,20 +1,18 @@
...
 import { Section } from "../../components/Section"
-import { ActionFooter } from "../../components/ActionFooter"
 import Link from "@docusaurus/Link"
 import { usePluralForm } from "@docusaurus/theme-common"
 import { translate } from "@docusaurus/Translate"
 import { MDXProvider } from "@mdx-js/react"
 import type { Props, Metadata } from "@theme/BlogPostPage"
 import MDXComponents from "@theme/MDXComponents"
 import Seo from "@theme/Seo"
-import EditThisPage from "@theme/EditThisPage"
```
Comment (argshook) [inline src/theme/BlogPostPage/index.tsx:12]: FYI, edit button was praised quite a bit once it was first added. I wouldn't remove it, perhaps find a nicer location for it, or maybe display just an icon

Comment (tris0laris) [reply src/theme/BlogPostPage/index.tsx:12]: thanks! I find the amount of space between the author and the tags quite significant, feels a bit empty in between. Wdyt ?

Comment (goodroot) [reply src/theme/BlogPostPage/index.tsx:12]: Ah yeah -- this is very much me learning the structure and proposing within elements. For now, looking for general feedback on the idea of the relocations and overall philosophy. The styling and more appropriate code formatting comes next if we think the direction is OK.

Comment (goodroot) [reply src/theme/BlogPostPage/index.tsx:12]: Iterated a bit. Getting there. Needs dark mode support, and some polish. 👍

File: blog/2023-08-18-enterprise-rbac-acl-tutorial.mdx [hunk 2]
```diff
@@ -18,23 +22,40 @@ import Screenshot from "@theme/Screenshot"
...
+For example, analysts, support agents and similar may just want to `SELECT` and
+read wherever appropriate and work with the peace-of-mind that they will not
+**blow something up** by mistake. While an overall administrator will want to
+alter permissions, see everything, create tables and `INSERT` data.
 
 If someone no longer needs access? One command and they are revoked.
 
 How does it work? Proceed!
 
 ## RBAC in QuestBD Enterprise
```
Comment (oliver-daniel) [inline blog/2023-08-18-enterprise-rbac-acl-tutorial.mdx:52]:
```suggestion
## RBAC in QuestDB Enterprise
```

Comment (goodroot) [reply blog/2023-08-18-enterprise-rbac-acl-tutorial.mdx]: 😆 Wow.

**Pull request merged**